### DISCUSSION TITLE
New mod idea: Bullseye

### DISCUSSION THREAD

**Discussion by FerasLR** [ppy/osu#22660] [Ideas]
The idea of the mod is rather than only judging the accuracy with hit timing, it would make it so that it judges it based on your hit position as well, hitting the edge would count as a 50, hitting the outer circle would count as a 100, and hitting the middle would count as a 300.

![image](https://user-images.githubusercontent.com/51180658/219327830-fb5b2ecf-88ce-491b-90df-6c58ff976b44.png)

this is a rough example of how the hit windows would behave like in a hitcircle.

**Comment by Rekunan** [comment]
I think we could take this further and create a sort of unstable rate for aim, using a [similar formula](https://osu.ppy.sh/wiki/en/Gameplay/Accuracy#accuracy) where instead it's the standard deviation of the aim error (probably in pixels).

**Reply by Axilotl17** [reply]
how would you get real-time feedback in terms of 100s and 50s?

**Reply by ItsShamed** [reply]
I think that Wieku was able to do an Aim UR in danser, it has even visualisation for aim error.

**Reply by Rekunan** [reply]
After experimenting with aim error in danser, this is definitely what I'm talking about, ty ItsShamed. If anyone wants to have a try, here's the code from danser: https://github.com/Wieku/danser-go/blob/master/app/states/components/overlays/play/aimerror.go

@Axilotl17 Honestly I was thinking of just adding aim error and stop there. If we continue further to add real-time feedback, we'd also have to consider adding an aim accuracy value, which I wouldn't be entirely opposed to, but that'd also mean wondering about past scores and how to even implement that in the game. Perhaps we could show two score details for aim and tapping? Or at least a new/separate section. If we do this, then probably just have it show two real-time judgements: aim and tapping.

**Reply by Axilotl17** [reply]
isnt this entire discussion about a mod? we would just have accuracy in terms of that mod, exactly as proposed in the original post? how would we have to consider older plays to that?

i think your idea about aim UR is a separate discussion.

**Reply by Rekunan** [reply]
Fair enough, that's my apologizes, although I believe there's still merit for having aim UR be implemented within this mod if possible.

I also think there's a disconnect with our interpretation, because the post says "rather than only... with hit timing" implying that accuracy would be both aim and tapping. If it's switched to be only aim then that's fair, but if it's both then there needs to be considerations of how both are displayed.

**Comment by mk56-spn** [comment]
not sure about this not fully against it but it does go counter to how people intuitively play, where you aim the least amount possible in most case, ie when theres a half stacked double people aim to hit both not one and then the other for the most part, thats all i have to add on the matter

**Reply by Axilotl17** [reply]
> where you aim the least amount possible in most case

I don't think anybody really plays like this at a high level, the example you provided is the one case I agree, but I think overall people aim to the middle of the circle.

**Reply by Rekunan** [reply]
Adding on, there's actually been a study on aim error, showing that players actually aim to the center (ignoring exceptions like the aforementioned case): https://docs.google.com/document/d/1ThW5OUfdsc63RJ6EBtEIfcb7CoWl7Yfaj_Emz9_aEZg/edit

**Comment by peppy** [comment]
I think target practice should incorporate this. Was kinda the original idea (in stable hitting the targets closer to the centre made them shatter better).

**Reply by Axilotl17** [reply]
yes but could it not be included without target practice?

**Reply by TheFearlessDeath500** [reply]
> yes but could it not be included without target practice?

no, since stable is deprecated

**Reply by Axilotl17** [reply]
???? not remotely what i'm asking. 

I think that this as a separate mod to change how accuracy works, much like strict tracking. target practice doesn't have sliders, and kind of butchers the map IIRC. having the ability to train aim accuracy in actual maps would be amazing.
